GliaWidgets/Resources/en.lproj/Localizable.strings

```diff
@@ -263,3 +263,5 @@
 "callVisualizer.screenSharing.accessibility.messageHint" = "Message label";
 "callVisualizer.screenSharing.accessibility.buttonLabel" = "End screen sharing";
 "callVisualizer.screenSharing.accessibility.buttonHint" = "Ends screen sharing";
+
+"gva_not_supported" = "This action is not currently supported on mobile";
```

Comment:
If you want to use dots, you can. Don't worry to much at the moment. What is important is that the words in the key are the same as in Android for now.

Reply:
Let's leave as is for now